marklogic-data-hub-central/ui/src/components/modeling/entity-type-modal/entity-type-modal.tsx

```diff
@@ -102,7 +135,7 @@ const EntityTypeModal: React.FC<Props> = (props) => {
         <Form.Item
           className={styles.formItem}
           label={<span>
-            Name:&nbsp;<span className={styles.asterisk}>*</span>
+            Name:&nbsp;{props.isEditModal ? null : <span className={styles.asterisk}>*</span>}
```

Comment:
Going by the wireframe, Edit is supposed to have the Entity Type name as text, not as a disabled input

Reply:
Good catch. Updated in here: https://github.com/marklogic/marklogic-data-hub/pull/4017/commits/e8d869cc937370f183c2574b9e61293edd943442